app/src/components/UI/Pages/Login.tsx

```diff
@@ -10,73 +10,147 @@ import {
 } from "../../../api";
 import { useAppSelector } from "../../../store";
 import { useNavigate } from "react-router-dom";
+import MessageError from "../Atoms/Message/MessageError";
+import { Formik, FormikProps, ErrorMessage } from "formik";
+import * as Yup from "yup";
 
 /**
  * Displays the Login element in the login page
  */
 
+interface myvalues {
+  Username: string;
+  Password: string;
+}
+
 export default function Login(): React.ReactElement {
-  const [credentials, setCredentials] = useState<Credentials>({
-    Password: "",
-    Username: "",
-  });
+  // const [credentials, setCredentials] = useState<Credentials>({
```

Comment:
If this is not necessary anymore, you should just delete it instead of commenting it.